docs/controls/risk_assessment/ra-05.md

```diff
@@ -0,0 +1,32 @@
+# Risk Assessment
+## RA-05 - VULNERABILITY SCANNING
+
+The organization:  
+a. Scans for vulnerabilities in the information system and hosted applications (Assignment: organization-defined frequency and/or randomly in accordance with organization-defined process) and when new vulnerabilities potentially affecting the system/applications are identified and reported;  
+
+b. Employs vulnerability scanning tools and techniques that facilitate interoperability among tools and automate parts of the vulnerability management process by using standards for:  
+   1. Enumerating platforms, software flaws, and improper configurations;  
+   2. Formatting checklists and test procedures; and  
+   3. Measuring vulnerability impact;  
+
+c. Analyzes vulnerability scan reports and results from security control assessments;  
+
+d. Remediates legitimate vulnerabilities (Assignment: organization-defined response times) in accordance with an organizational assessment of risk; and  
+
+e. Shares information obtained from the vulnerability scanning process and security control assessments with (Assignment: organization-defined personnel or roles) to help eliminate similar vulnerabilities in other information systems (i.e., systemic weaknesses or deficiencies).  
+
+### TDP Implementation
+
+a. As part of the TDP Test Plan, security scans are completed on an ongoing basis, throughout the Continuous Integration (CI).  Automated scans are run on every push, pull request and merge on GitHub.  
+
+b. Security scanning is completed using OWASP ZAP dynamic security scans and Snyk vulnerability dependency scanning.  Snyk will automatically open Pull Requests if there is a vulnerability dependency.  If there are no findings, no Pull Requests will be opened.  OWASP Zap scans are the last step for each CI run.  The results for the scans are summarized and accessed through CircleCI.  If vulnerabilities are found in the scan results, the code will be prevented from being deployed until the vulnerabilities are remediated.
+
+c. Summaries of the security scan reports are reviewed in CircleCI. (see screenshot of summaries of scan reports below)
+
+![screenshot - Summaries of security scan reports](https://github.com/raft-tech/TANF-app/blob/documentation/risk-assessment-ra-5/docs/controls/risk_assessment/ra-5-screenshots/Screen%20Shot%202021-01-29%20at%203.05.42%20PM.png)
```

Comment:
@rnafarrete Small non-content change: let's use relative links when referencing screenshots so that they refer directly to the file contained in this repo rather than an external repo (`raft-tech` in this case.

Reply:
Changed to relative link.